modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -318,6 +339,13 @@ public CacheMetricsImpl(GridCacheContext<?, ?> cctx, boolean isNear) {
 
             return fut != null && !fut.isDone();
         }, "True if index rebuild is in progress.");
+
+        getHistogram = mreg.histogram("GetHistogram", HISTOGRAM_BUCKETS, "Get duration histogram in nanoseconds.");
+
+        putHistogram = mreg.histogram("PutHistogram", HISTOGRAM_BUCKETS, "Put duration histogram in nanoseconds.");
```

Comment:
nit: the word "histogram" in the description is useless. Let's remove it.

Reply:
Done.